src/services/helper/sessions.js

```diff
@@ -426,6 +430,19 @@ module.exports = class SessionsHelper {
 						}
 					})
 				)
+				const userIds = sessionDetails[0].data
+					.map((item) => item.userId.toString())
+					.filter((value, index, self) => self.indexOf(value) === index)
+
+				let mentorDetails = await UserProfileHelper.getAllMentorsDetail(userIds)
```

Comment:
@ankitpws change to getListOfUserDetails in enhancement

Reply:
@ankitpws we don't require all the fields, can we pass only the required fields from here, take it enahncement